etcdctl/ctlv3/command/get_command.go

```diff
@@ -19,7 +19,9 @@ import (
 	"strings"
```

Comment:
I see. As you commented, I'll drop the etcdctl part in this PR and revisit it later. Providing the soft-fail mode for other operations for checking write permissions would be valuable.